sdk/eventgrid/azure-mgmt-eventgrid/CHANGELOG.md

```diff
@@ -1,5 +1,45 @@
 # Release History
 
+## 3.0.0rc5 (2020-04-01)
```

Comment:
i changed version here to 3.0.0.rc5 (instead of 4.0.0rc1)
as rc4 was preview / rc version, no need to update major version, even if we have breaking
There's missing update in version file.